### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by carusogabriel**
I've improved our suite of tests, by:
- Adding `PHPUnit 6` support;
- Refactoring some tests with `assertEmpty` method.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "php": "^5.3.2 || ^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.5 || ^5.0.5"
+        "phpunit/phpunit": "^4.8.35 || ^5.7 || ^6.5"
```
Comment (johnstevenson) [inline composer.json]: Is there any reason for the specific patch version (ie, why not 4.8.36, which is the latest). Wouldn't ^4.8 suffice?

File: tests/XdebugHandlerTest.php [hunk 2]
```diff
@@ -121,7 +122,7 @@ public function testEnvAllowWithEmptyScanDir()
         putenv('PHP_INI_SCAN_DIR');
         $xdebug = new XdebugHandlerMock($loaded);
         $xdebug->check();
-        $this->assertEquals('', getenv('PHP_INI_SCAN_DIR'));
+        $this->assertEmpty(getenv('PHP_INI_SCAN_DIR'));
```
Comment (johnstevenson) [inline tests/XdebugHandlerTest.php]: This, and the other `assertEquals('', ...)` tests are testing for a string.

File: composer.json (hunk 1 above)
Comment (carusogabriel) [reply composer.json]: 4.8.35 was the first PHPUnit 4.* version to support namespaces

File: tests/XdebugHandlerTest.php (hunk 2 above)
Comment (carusogabriel) [reply tests/XdebugHandlerTest.php]: Gonna rollback this changes soo

File: composer.json (hunk 1 above)
Comment (johnstevenson) [reply composer.json]: Ok, thanks for clarifying

File: tests/XdebugHandlerTest.php (hunk 2 above)
Comment (johnstevenson) [reply tests/XdebugHandlerTest.php]: Thanks. If you use `assertSame`, rather than just revert the changes then you will fix a bug.

**Comment by carusogabriel** [comment]
I've used `assertSame` instead of `assertEquals`, and also, [here is the changelog for `PHPUnit 4.8`](https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-4.8.md#4835---2017-02-06) that supports the new namespace `PHPUnit\Framework\TestCase`.

File: tests/XdebugHandlerTest.php [hunk 3]
```diff
@@ -121,7 +122,7 @@ public function testEnvAllowWithEmptyScanDir()
         putenv('PHP_INI_SCAN_DIR');
```
Comment (johnstevenson) [inline tests/XdebugHandlerTest.php]: Cool, you've fixed a bug and found one! The env is already correctly set at the top of the test, so please remove this line and replace the comment above it, so:

```php
    
    // Mimic restart
    $xdebug = new XdebugHandlerMock($loaded);
```

Comment (carusogabriel) [reply tests/XdebugHandlerTest.php]: Now everything is working. Sorry for the mess :sweat_smile:

Comment (johnstevenson) [reply tests/XdebugHandlerTest.php]: Many thanks for sorting it.

**Pull request merged**